Update README.md

deniz-lktr opened the pull request:
Clarify the only working version currently supported is v5.x. The readme can give the impression v5+ is supported as the instructions tell you to use 'latest'.

deniz-lktr committed:
Update README.md

Clarify the only working version currently supported is v5.x. The readme can give the impression v5+ is supported as the instructions tell you to use 'latest'.

TheSecurityDev commented:
Good catch, but maybe you should also change or just remove the installation instructions which tell you to install the latest version.

deniz-lktr committed:
Update instructions to set a fixed package version

simple-koa-shopify-auth (currently) only supports v5.x, which is
no longer being actively developed. Clarify the latest version
in the comments to avoid misunderstandings.

deniz-lktr commented:
On this note, v5 has several issues that makes it hard (impossible) to migrate. https://github.com/Shopify/shopify-api-js/issues/400 and https://github.com/Shopify/shopify-api-js/issues/605 to name a few. It would be great if this library was upgraded to support v6+ 😄
